backend/tests/unit/dal/Services/ProjectServiceTest.cs

```diff
@@ -414,6 +414,8 @@ public void Add_Project_SimpleFields()
             project.ProjectNumber = "test-generation-override";
             project.PrivateNote = "private note";
             project.PublicNote = "public note";
+            project.ExemptionRequested = true;
+            project.ExemptionRational = "Providing reasoning for exemption request";
```

Comment:
/nit: do we want `ExemptionRational` or `ExemptionRationale`?

Reply:
Fixing now, was just trying to avoid DB refresh but I need to satisfy my OCD lol